### PR TITLE
FAT-1568 - https://issues.folio.org/browse/FAT-1568

### PR DESCRIPTION
**Pull request by gurleenkaurbp**
Purpose/Overview:
Karate implementation is created to test the following scenario via edge module:

Can create patron hold transaction and related FOLIO circulation/inventory records

Result -
image

### CODE REVIEW

File: edge-inn-reach/src/test/java/org/folio/EdgeInnReachApiTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        runFeatureTest("inn-reach-proxy.feature");
+    }
+
+    @Override
+    public void runHook() {
+        super.runHook();
+        System.setProperty("testTenant", "diku");
+    }
+
+//    @BeforeAll
```
Comment (Arghya22) [inline edge-inn-reach/src/test/java/org/folio/EdgeInnReachApiTest.java:34]: add BeforeAll

File: edge-inn-reach/src/main/resources/volaris/edge-inn-reach/features/inn-reach-proxy.feature [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#@ignore
```
Comment (Arghya22) [inline edge-inn-reach/src/main/resources/volaris/edge-inn-reach/features/inn-reach-proxy.feature:1]: add @Ignore

**Review by Arghya22 — COMMENTED** [review]
Please check the below points

File: edge-inn-reach/src/test/java/org/folio/EdgeInnReachApiTest.java (hunk 1 above)
Comment (gurleenkaurbp) [reply edge-inn-reach/src/test/java/org/folio/EdgeInnReachApiTest.java]: Added.

File: edge-inn-reach/src/main/resources/volaris/edge-inn-reach/features/inn-reach-proxy.feature (hunk 2 above)
Comment (gurleenkaurbp) [reply edge-inn-reach/src/main/resources/volaris/edge-inn-reach/features/inn-reach-proxy.feature]: Done

**Review by Arghya22 — APPROVED** [review]

**Review by arin-epm — APPROVED** [review]

**Pull request merged**